packages/vscode-extension/src/parameterService.ts

```diff
@@ -36,6 +37,7 @@ export function getParameters(): Parameters {
 export async function clearParameters(): Promise<void> {
   const context = getExtensionContext();
   await context.globalState.update(PARAMETERS_KEY, {});
+  await sendParametersToLanguageServer();
```

Comment:
This was a bug, when clearing the parameters those should go out of scope for the language server.